src/handlers.rs

```diff
@@ -244,6 +252,69 @@ fn handle_code_cache(
   Ok(null_buf()) // null response indicates success.
 }
 
+fn handle_set_env(
+  d: *const DenoC,
+  _builder: &mut FlatBufferBuilder,
+  key: &str,
+  value: &str,
+) -> HandlerResult {
+  let deno = from_c(d);
+  if !deno.flags.allow_env {
+    let err = std::io::Error::new(
+      std::io::ErrorKind::PermissionDenied,
+      "allow_env is off.",
+    );
+    return Err(err.into());
+  }
+
+  std::env::set_var(key, value);
+  Ok(null_buf())
+}
+
+fn handle_env(
+  d: *const DenoC,
+  builder: &mut FlatBufferBuilder,
+) -> HandlerResult {
+  let deno = from_c(d);
+  if !deno.flags.allow_env {
+    let err = std::io::Error::new(
+      std::io::ErrorKind::PermissionDenied,
+      "allow_env is off.",
+    );
+    return Err(err.into());
+  }
+
+  let vars: Vec<_> = std::env::vars().map(|(key, value)| {
+    let key = builder.create_string(&key);
+    let value = builder.create_string(&value);
+
+    msg::Entry::create(builder, &msg::EntryArgs {
+      key: Some(key),
+      value: Some(value),
+      ..Default::default()
+    })
+  }).collect();
+
+  let offsets = builder.create_vector_of_reverse_offsets(&vars);
```

Comment:
This probably doesn't matter, since we're going to read all of the values anyway, but [this page](https://github.com/google/flatbuffers/blob/master/docs/source/CppUsage.md#storing-maps--dictionaries-in-a-flatbuffer) suggest using CreateVectorOfSortedTables ... What do you think?

Reply:
@ry I read that, and I couldn't find that method looking at the source of `flatbuffers` in `third_party`.